scame/__main__.py

```diff
@@ -189,7 +189,7 @@ def _git_diff_files(ref="master"):
         print("Failed to diff files.")
         sys.exit(1)
 
-    for line in output.splitlines():
+    for line in output.decode("utf-8").splitlines():
```

Comment:
this is ok... a better fix would have been to update `_execute` to always return `str/unicode` and not bytes... or at least return unicode by default.

I don't think that we have any use case in which we want to receive the stdout as raw data in scame.